Cargo.toml

```diff
@@ -21,6 +21,7 @@ plc_source = { path = "./compiler/plc_source" }
 plc_ast = { path = "./compiler/plc_ast" }
 plc_util = { path = "./compiler/plc_util" }
 plc_diagnostics = { path = "./compiler/plc_diagnostics" }
+plc_index = { path = "compiler/plc_index" }
```

Comment:
nitpick: is there a reason this is not similar to the crates above? ¨./...¨

Reply:
No reason, fixed 👍